noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/TwitterFeedUsersTimelineDriver.java

```diff
@@ -1,31 +1,43 @@
 package edu.uci.ics.cloudberry.noah.feed;
 
+import com.twitter.hbc.ClientBuilder;
+import com.twitter.hbc.core.Client;
+import com.twitter.hbc.core.Constants;
+import com.twitter.hbc.core.endpoint.StatusesFilterEndpoint;
+import com.twitter.hbc.core.processor.StringDelimitedProcessor;
+import com.twitter.hbc.httpclient.auth.Authentication;
+import com.twitter.hbc.httpclient.auth.OAuth1;
 import org.kohsuke.args4j.CmdLineException;
 import twitter4j.*;
 import twitter4j.conf.ConfigurationBuilder;
 
 import java.io.BufferedWriter;
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
 
 public class TwitterFeedUsersTimelineDriver {
 
     public void run(Config config) throws IOException {
+        List<Long> usersID = new ArrayList<Long>();
 
         ConfigurationBuilder builder = new ConfigurationBuilder();
-        builder.setDebugEnabled(true)
+        builder.setDebugEnabled(false)
                 .setOAuthConsumerKey(config.getConsumerKey())
                 .setOAuthConsumerSecret(config.getConsumerSecret())
                 .setOAuthAccessToken(config.getToken())
                 .setOAuthAccessTokenSecret(config.getTokenSecret())
                 .setJSONStoreEnabled(true);
-
+        //Get historical user data
```

Comment:
whenever you need the comments to describe the code, that is the time to create a separate function to wrap up the corresponding code. 

there should be a `getHistoricalUserData` function.